.NET/Microsoft.Recognizers.Definitions.Common/Italian/NumbersDefinitions.cs

```diff
@@ -16,367 +16,458 @@
-            { @"cento", 100 },
-            { @"due cento", 200 },
-            { @"duecento", 200 },
-            { @"tre cento", 300 },
-            { @"trecento", 300 },
-            { @"quattro cento", 400 },
-            { @"quattrocento", 400 },
-            { @"cinque cento", 500 },
-            { @"cinquecento", 500 },
-            { @"sei cento", 600 },
-            { @"seicento", 600 },
-            { @"sette cento", 700 },
-            { @"settecento", 700 },
-            { @"otto cento", 800 },
-            { @"ottocento", 800 },
-            { @"nove cento", 900 },
-            { @"novecento", 900 }
-        };
-      public static readonly Dictionary<string, long> SuffixOrdinalMap = new Dictionary<string, long>
-        {
-            { @"mille", 1000 },
-            { @"milione", 1000000 },
-            { @"miliardo", 1000000000 }
-        };
-      public static readonly Dictionary<string, long> RoundNumberMap = new Dictionary<string, long>
-        {
-            { @"cento", 100 },
-            { @"mille", 1000 },
-            { @"milione", 1000000 },
-            { @"milioni", 1000000 },
-            { @"miliardo", 1000000000 },
-            { @"miliardi", 1000000000 },
-            { @"trilione", 1000000000000 },
-            { @"trilioni", 1000000000000 },
-            { @"centinaio", 100 },
-            { @"centinai", 100 },
-            { @"centinaie", 100 },
-            { @"millesimo", 1000 },
-            { @"milionesimo", 1000000 },
-            { @"miliardersimo", 1000000000 },
-            { @"trilioneisimo", 1000000000000 },
-            { @"millesima", 1000 },
-            { @"milionesima", 1000000 },
-            { @"miliardersima", 1000000000 },
-            { @"trilioneisima", 1000000000000 },
-            { @"millesimi", 1000 },
-            { @"milionesimi", 1000000 },
-            { @"miliardersimi", 1000000000 },
-            { @"trilioneisimi", 1000000000000 },
-            { @"millesime", 1000 },
-            { @"milionesime", 1000000 },
-            { @"miliardersime", 1000000000 },
-            { @"trilioneisime", 1000000000000 },
-            { @"centinaia", 100 },
-            { @"migliaia", 1000 },
-            { @"milionata", 1000000 },
-            { @"miliardata", 1000000000 },
-            { @"trilionata", 1000000000000 },
-            { @"dozzina", 12 },
-            { @"dozzine", 12 },
-            { @"k", 1000 },
-            { @"m", 1000000 },
-            { @"g", 1000000000 },
-            { @"b", 1000000000 },
-            { @"t", 1000000000000 }
-        };
-      public static readonly Dictionary<string, string> RelativeReferenceOffsetMap = new Dictionary<string, string>
-        {
-            { @"", @"" }
-        };
-      public static readonly Dictionary<string, string> RelativeReferenceRelativeToMap = new Dictionary<string, string>
-        {
-            { @"", @"" }
-        };
-    }
+	public static class NumbersDefinitions
+	{
+		public const string LangMarker = @"Ita";

```

Comment:
This indentation is all over the place. Please double check that all the generated resource files were generated using the latest code from /master and that any changes includes on the PR are actually purposeful, ok?
Otherwise the whole review process becomes much harder.